model/fv_arrays.F90

```diff
@@ -361,9 +361,15 @@ module fv_arrays_mod
    logical :: do_sat_adj= .false.   !< Controls split GFDL Microphysics. .false. by default. Must have the same
```

Comment:
We may wish to eventually move these options related to the inline physics to a new namelist. fv_core_nml is already way too long.

Reply:
@lharris4 is this a suggestion to be addressed now? Or would you like it addressed in a future release?

Reply:
I can do this in a future release.